src/bias/PBMetaD.cpp

```diff
@@ -33,6 +33,7 @@
 #include "tools/File.h"
 #include <ctime>
 #include <numeric>
+#include <iostream>
```

Comment:
I think this could be removed if iostream objects are not explicitly used in the code (see comment below)

Reply:
Right, removed.